test/e2e/cypress/fixtures/hosts-overview/hosts_overview.feature

```diff
@@ -0,0 +1,98 @@
+Feature: Hosts Overview
+This is where the user has an overview of the status of all the hosts in the deployed SAP system
+
+Background:
+    Given an healthy SAP deployment of 27 hosts having following agent ids and hostnames
+    # a09d9cf3-46c1-505c-8fb8-4b0a71a9114e => vmdrbdprd01
```

Comment:
Is it a good idea to have all the entries here?
If for some reason we change some file, this won't match. But maybe there is not any change to doing so

Reply:
The entries are just a reference here, even if we change something, that would not break anything, there will be a little inconsistency that would be easy to fix and doesn't harm that much, I think.